web/data/accounts.go

```diff
@@ -155,6 +156,21 @@ func createAccount(c echo.Context) error {
 	account.Encrypt(doc)
 	account.ComputeName(doc)
 
+	if doc.M["cozyMetadata"] == nil {
+		cm := metadata.New()
+
+		data, err := json.Marshal(cm)
+		if err != nil {
+			return err
+		}
+
+		doc.M["cozyMetadata"] = new(map[string]interface{})
+		err = json.Unmarshal(data, doc.M["cozyMetadata"])
+		if err != nil {
+			return err
+		}
```

Comment:
Why not just `doc.M["cozyMetadata"] = metadata.New()`?

Reply:
Well, we expect `doc.M["cozyMetadata"]` to be of type `map[string]interface{}` while `metadata.New()` returns a type `*CozyMetadata`.
Maybe that's overkill since we save `doc` right after that but I wanted to stay consistent with the usage we have of this attribute in other places (e.g. https://github.com/cozy/cozy-stack/blob/master/model/sharing/sharing.go#L1702)